dcm2niix/prepare.py

```diff
@@ -0,0 +1,42 @@
+"""Function to execute setup for dcm2niix."""
+
+import glob
+import logging
+import os
+
+import pydicom
+
+from dcm2niix import arrange
+from dcm2niix import dcm2niix_utils
+
+
+log = logging.getLogger(__name__)
+
+
+def setup(infile, rec_infile, work_dir, remove_incomplete_volumes, decompress_dicoms):
+    """Prepare dcm2niix input, remove incomplete volumes, and decompress dicom files."""
+    log.info("Prepare dcm2niix input.")
+    dcm2niix_input_dir = arrange.prepare_dcm2niix_input(infile, rec_infile, work_dir)
+
+    if remove_incomplete_volumes:
+        log.info("Remove incomplete volumes.")
+        dicom_file = glob.glob(dcm2niix_input_dir + "/*")[0]
+
+        try:
+            pydicom.filereader.dcmread(dicom_file)
```

Comment:
`force=True` to allow for bypassing Dicom signature check?

Reply:
Changed.